package/test/daml/Daml.Finance.Util.Test/daml.yaml

```diff
@@ -12,8 +12,8 @@ dependencies:
 data-dependencies:
   - .lib/daml-finance/Daml.Finance.Interface.Types.Common/1.0.1/daml-finance-interface-types-common-1.0.1.dar
   - .lib/daml-finance/Daml.Finance.Interface.Types.Date/2.0.1/daml-finance-interface-types-date-2.0.1.dar
-  - .lib/daml-finance/Daml.Finance.Interface.Util/1.0.1/daml-finance-interface-util-1.0.1.dar
+  - .lib/daml-finance/Daml.Finance.Interface.Util/2.0.1/daml-finance-interface-util-2.0.1.dar
   - .lib/daml-finance/Daml.Finance.Test.Util/1.0.0/daml-finance-test-util-1.0.0.dar
-  - .lib/daml-finance/Daml.Finance.Util/2.0.1/daml-finance-util-2.0.1.dar
-
+  - .lib/daml-finance/Daml.Finance.Util/3.0.1/daml-finance-util-3.0.1.dar
+build-options: null
```

Comment:
remove `build-options`

Reply:
packell adds them so I'm afraid someone's commit will at some point add them back anyway

Reply:
I just checked and `build-options: null` is in basically all packages. I will leave it for now, we can remove it once `packell` is updated accordingly